docs/source/tracking.rst

```diff
@@ -433,3 +433,39 @@ then make API requests to your remote tracking server.
+| Key                           | Description                                                                            |
++===============================+========================================================================================+
+| ``mlflow.runName``            | Human readable name that identifies this run.                                          |
++-------------------------------+----------------------------------------------------------------------------------------+
+| ``mlflow.parentRunId``        | The ID of the parent run, if this is a nested run.                                     |
++-------------------------------+----------------------------------------------------------------------------------------+
+| ``mlflow.user``               | Identifer of user who created the run.                                                 |
++-------------------------------+----------------------------------------------------------------------------------------+
+| ``mlflow.source.type``        | Source type (possible values are ``"NOTEBOOK"``, ``"JOB"``, ``"PROJECT"``,             |
+|                               | ``"LOCAL"``, and ``"UNKNOWN"``)                                                        |
++-------------------------------+----------------------------------------------------------------------------------------+
+| ``mlflow.source.name``        | Source identifier (e.g., GitHub URL, local Python filename, name of notebook           |
++-------------------------------+----------------------------------------------------------------------------------------+
+| ``mlflow.source.git.commit``  | Commit hash of the executed code, if in a git repository.                              |
++-------------------------------+----------------------------------------------------------------------------------------+
+| ``mlflow.source.git.branch``  | Name of the branch of the executed code, if in a git repository.                       |
++-------------------------------+----------------------------------------------------------------------------------------+
+| ``mlflow.source.git.repoURL`` | URL that the executed code was cloned from.                                            |
++-------------------------------+----------------------------------------------------------------------------------------+
+| ``mlflow.project.env``        | One of "docker" or "conda", indicating the runtime context used by the mlflow project. |
++-------------------------------+----------------------------------------------------------------------------------------+
+| ``mlflow.project.entryPoint`` | Name of the project entry point associated with the current run, if any.               |
++-------------------------------+----------------------------------------------------------------------------------------+
+| ``mlflow.docker.image.name``  | Name of the Docker image used to execute this run.                                     |
```

Comment:
As per offline conversation: Reminder to add context that his refers to docker "Projects",
Also applies to `mlflow.docker.image.id`